### ISSUE TITLE
Cucumber snippets listed in cucumber.js should be ignored

### ISSUE THREAD

**Issue by kevgo** [charlierudolph/dependency-lint#27]
I am using https://github.com/kevgo/cucumber-snippets-livescript to provide Cucumber snippets in Livescript. Those show up as false positive unused modules. I think a simple match for:

```
--snippet-syntax node_modules/XXX"
```

should add XXX to the list of automatically ignored modules.

**Comment by charlierudolph** [comment]
Hmm. Thats a tough case. I'm not certain how to deal with that generically. I guess we could search for `node_modules/*` or `./node_modules/*` in the npm scripts and shell scripts. What do you think of that?

**Comment by kevgo** [comment]
Maybe a super-generic `executedModules/other` category, which just does a regex search for
 `\b node_modules/(.*)\b` in the files listed there?

**Comment by charlierudolph** [comment]
I don't think this belongs under `executedModules` but maybe something new called `referencedModules` which can either do a search for `node_modules/<name>` or just `<name>` in some configured files

**Comment by kevgo** [comment]
Yes, `referencedModules` sounds like the right place for this.
